core/services/log/broadcaster_test.go

```diff
@@ -86,18 +85,16 @@ func TestBroadcaster_ResubscribesOnAddOrRemoveContract(t *testing.T) {
 	var backfillCount int64
 	var backfillCountPtr = &backfillCount
 
-	// the first backfill should use the last known height from db
+	// the first backfill should use the last known height from db minus 1 (num confirmations of subscriber)
```

Comment:
Do we have a test covering the off-by-one case?

Reply:
hm i'm not sure we can call it off-by-one. I made it more explicit now by defining the 1 in 'numConfirmations' variable